StudentProject/StudentDetails.java

```diff
@@ -48,88 +27,86 @@ public static void main(String[] args) {
             ch = s.nextInt();
 
             switch (ch) {
-                case 1:
+                case ADD_STUDENT:
                     System.out.println("--------------------------------------------------");
                     System.out.println("Please provide the data for the student to store. ");
                     System.out.print("Student id: ");
-                    int sno = s.nextInt();
+                    int eno = s.nextInt();
                     System.out.print("First name: ");
-                    String sfname = s1.nextLine();
+                    String StudentFirstName = s1.nextLine();
                     System.out.print("Last name: ");
-                    String slname = s1.nextLine();
-
-                    c.add(new StudentData(sno, sfname, slname));
+                    String StudentLastName = s1.nextLine();
+                    c.add(new StudentData(eno, StudentFirstName, StudentLastName));
                     break;
-                case 2:
+                case SHOW_STUDENT:
                     System.out.println("--------------------------------------------------");
-                    System.out.println("Id fname lname");
+                    System.out.printf("%-10s %-15s %-15s", "id", "firstname", "lastname");
+                    System.out.println();
                     Iterator<StudentData> i = c.iterator();
```

Comment:
Instead of an Iterator, you can also use a for each loop. for(Student currentStudent : c) { ... }